server/public/index.html

```diff
@@ -3,8 +3,13 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Final Project</title>
+  <title>Not Etsy</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css"
```

Comment:
I recommend using Bootstrap `4.5`. It's the most up-to-date version of Bootstrap 4.

https://getbootstrap.com/docs/4.5/getting-started/introduction/